api/api_calls.py

```diff
@@ -505,7 +505,7 @@ def send_sequence_files(self, sequence_file, sample_name, project_id, upload_id)
         read_size = 32768
         self._stop_upload = False
 
-        def _send_file(filename, parameter_name, bytes_read=0):
```

Comment:
Reworked this function to not use `bytes_read` because we are monitoring upload per file, not per file pairs now.